### ISSUE TITLE
Reved <img> paths not updated in all html files

### ISSUE THREAD

**Issue by kermit666** [yeoman/generator-angular#1049]
The html files in *dist/* created with `grunt build` are not updated to match the reved image file names. For example in an html file at path *dist/views/blocks/aside.right.html*, a tag remained as:

    <img src="images/a4.jpg" alt="..." class="img-circle">

While in the dist/images/ folder this file was named `a4.7cd2fdaf.jpg`.

Am I doing something wrong? The html file is included using `ng-include`. Is that why its not processed by grunt? I thought that this usemin rule covers it:

```
    usemin: {
      html: ['<%= yeoman.dist %>/{,*/}*.html'],
      css: ['<%= yeoman.dist %>/styles/{,*/}*.css'],
      options: {
        assetsDirs: [
          '<%= yeoman.dist %>',
          '<%= yeoman.dist %>/images',
          '<%= yeoman.dist %>/styles'
        ]
      }
    },
```

My installed version is:

```
npm list -g | grep angular
├─┬ generator-angular@0.11.1
```

**Comment by sindresorhus** [comment]
This is already clearly explained in your Gruntfile: https://github.com/yeoman/generator-angular/blob/e792915e07590dbc3d8d79e1df4311444571084a/templates/common/root/_Gruntfile.js#L4-L8

**Comment by kermit666** [comment]
Oh, missed that comment. When I tried switching to `**` for globbing of html files (see the [diff](https://gist.github.com/kermit666/efc1304e88392fff408c)), I still got the problem of some resources not being available like main.js, app.js, main.css... For some reason, they were reved and the index.html didn't reflect it.

```
$ ls dist/scripts
scripts.69f34af7.js vendor.171d9a80.js
```

I ended up just flattening out the files in the views/ folder to avoid this issue.
